test/pairwise_test.js

```diff
@@ -0,0 +1,47 @@
+var testFile = require('../challenges/pairwise.js');
```

Comment:
while this code is fine, I feel it would be better organized if the 'test' subfolder were put into the challenges subfolder.  Since this project has two different things in it (challenges and projects/markdown-prev,  it is hard to interpret if the top level 'test' folder goes with all of them or just one or the other.